source/compatibility.tex

```diff
@@ -64,7 +64,7 @@
 \diffref{lex.string}
 \change
 String literals made const.\\
-The type of a string literal is changed
+The type of a \grammarterm{string-literal} is changed
```

Comment:
(Digression: I think we are presenting *string-literal*s very strangely. We should have something like:

> *string-literal-token*:
> &nbsp; &nbsp; \<current *string-literal* productions>

> *string-literal*:
> &nbsp; &nbsp; *string-literal-token*
> &nbsp; &nbsp; *string-literal* *string-literal-token*

And we should delete phase 6 of translation and move the string literal concatenation and string literal objects wording out of [lex] into [expr]. Then we could clearly distinguish between the cases where we're talking about *string-literal*s, such as this, and the cases where we're talking about *string-literal-token*s, which covers a bunch of the other changes in this PR.)

Reply:
Uh, is introducing new grammar-terms and deleting a phase of translation considered editorial these days?

Reply:
No, hence the "digression" part. :) I think that is something we should take to CWG for the C++23 cycle.